crates/core/tedge/src/cli/connect/command.rs

```diff
@@ -200,8 +202,10 @@ pub fn bridge_config(
         }
         Cloud::Aws => {
             let params = BridgeConfigAwsParams {
-                connect_url: config.aws.url.or_config_not_set()?.clone(),
-                mqtt_tls_port: MQTT_TLS_PORT,
+                mqtt_host: HostPort::<MQTT_TLS_PORT>::try_from(
+                    config.aws.url.or_config_not_set()?.as_str(),
+                )
+                .map_err(TEdgeConfigError::from)?,
```

Comment:
I'm surprised to see a change for AWS and the same for Azure, but nothing for Cumulocity. Can this be the root cause of all the system tests failing to connect Cumulocity?

Reply:
The problem was with the string representation of the `HostPort` type, which I've fixed now to include the `port` value as well.